packages/stargate/src/queries/bank.ts

```diff
@@ -0,0 +1,57 @@
+import { Bech32, toAscii } from "@cosmjs/encoding";
+
+import { cosmos } from "../generated/codecimpl";
+import { QueryClient } from "../queryclient";
+
+/**
+ * Use this to convert a protobuf.js class to the interface (e.g. Coin to ICoin)
+ * in a ways that makes Jasmine's toEqual happy.
+ */
+function toObject<I extends object>(thing: I): Omit<I, never> {
```

Comment:
This is only needed for our test code? Or does it serve any purpose in production code as well?

Reply:
It normalizes all protobuf classes to objects with the same properties and values. We need it for testing but it is nice in general to get consist runtime types.